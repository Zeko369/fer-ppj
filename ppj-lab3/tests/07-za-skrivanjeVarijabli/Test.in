<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 n
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       BROJ 1 10
      <T_lista>
       $
     <E_lista>
      $
  <lista_naredbi>
   <naredba>
    <naredba_pridruzivanja>
     IDN 2 rez
     OP_PRIDRUZI 2 =
     <E>
      <T>
       <P>
        BROJ 2 0
       <T_lista>
        $
      <E_lista>
       $
   <lista_naredbi>
    <naredba>
     <naredba_pridruzivanja>
      IDN 3 i
      OP_PRIDRUZI 3 =
      <E>
       <T>
        <P>
         BROJ 3 15
        <T_lista>
         $
       <E_lista>
        $
    <lista_naredbi>
     <naredba>
      <za_petlja>
       KR_ZA 4 za
       IDN 4 i
       KR_OD 4 od
       <E>
        <T>
         <P>
          BROJ 4 1
         <T_lista>
          $
        <E_lista>
         $
       KR_DO 4 do
       <E>
        <T>
         <P>
          IDN 4 n
         <T_lista>
          $
        <E_lista>
         $
       <lista_naredbi>
        <naredba>
         <naredba_pridruzivanja>
          IDN 5 rez
          OP_PRIDRUZI 5 =
          <E>
           <T>
            <P>
             IDN 5 rez
            <T_lista>
             $
           <E_lista>
            OP_PLUS 5 +
            <E>
             <T>
              <P>
               IDN 5 i
              <T_lista>
               OP_PUTA 5 *
               <T>
                <P>
                 IDN 5 i
                <T_lista>
                 OP_PUTA 5 *
                 <T>
                  <P>
                   IDN 5 i
                  <T_lista>
                   $
             <E_lista>
              $
        <lista_naredbi>
         <naredba>
          <naredba_pridruzivanja>
           IDN 6 x
           OP_PRIDRUZI 6 =
           <E>
            <T>
             <P>
              BROJ 6 5
             <T_lista>
              $
            <E_lista>
             $
         <lista_naredbi>
          $
       KR_AZ 7 az
     <lista_naredbi>
      <naredba>
       <naredba_pridruzivanja>
        IDN 8 i
        OP_PRIDRUZI 8 =
        <E>
         <T>
          <P>
           IDN 8 i
          <T_lista>
           $
         <E_lista>
          OP_PLUS 8 +
          <E>
           <T>
            <P>
             BROJ 8 1
            <T_lista>
             $
           <E_lista>
            $
      <lista_naredbi>
       $
