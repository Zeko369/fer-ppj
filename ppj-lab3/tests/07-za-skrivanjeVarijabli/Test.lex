IDN 1 n
OP_PRIDRUZI 1 =
BROJ 1 10
IDN 2 rez
OP_PRIDRUZI 2 =
BROJ 2 0
IDN 3 i
OP_PRIDRUZI 3 =
BROJ 3 15
KR_ZA 4 za
IDN 4 i
KR_OD 4 od
BROJ 4 1
KR_DO 4 do
IDN 4 n
IDN 5 rez
OP_PRIDRUZI 5 =
IDN 5 rez
OP_PLUS 5 +
IDN 5 i
OP_PUTA 5 *
IDN 5 i
OP_PUTA 5 *
IDN 5 i
IDN 6 x
OP_PRIDRUZI 6 =
BROJ 6 5
KR_AZ 7 az
IDN 8 i
OP_PRIDRUZI 8 =
IDN 8 i
OP_PLUS 8 +
BROJ 8 1
