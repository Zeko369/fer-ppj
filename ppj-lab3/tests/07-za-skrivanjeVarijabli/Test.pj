n = 10 
rez = 0
i = 15
za i od 1 do n
 rez = rez + i*i*i 
 x = 5
az
i = i + 1
