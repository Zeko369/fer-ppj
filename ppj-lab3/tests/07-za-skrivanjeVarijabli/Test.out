4 1 n
5 2 rez
5 4 i
5 4 i
5 4 i
8 3 i
